src/console_commands.cpp

```diff
@@ -58,14 +55,36 @@ ConsoleCommands::ConsoleCommands(p2pool* pool)
 	}
 	m_shutdownAsync.data = this;
 
-	err = uv_tty_init(&m_loop, &m_tty, 0, 1);
-	if (err) {
-		LOGERR(1, "uv_tty_init failed, error " << uv_err_name(err));
+	uv_handle_type stdin_type = uv_guess_handle(0);
+	LOGINFO(3, "uv_guess_handle returned " << (int)stdin_type);
```

Comment:
This should be `static_cast<int>(stdin_type)`. I generally try to avoid C-style casts in C++ code.